Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.yml

```diff
@@ -124,3 +136,5 @@ script:
   subtype: python3
   type: python
 fromversion: 5.0.0
+tests:
+- No tests
```

Comment:
there is a test playbook

Reply:
No mentioned in the conf.json
Added test playbook that was found on the same pack: Packs/MicrosoftGraphAPI/TestPlaybooks/Microsoft_Graph_Generic_-_Test.yml